renovate.json

```
@@ -0,0 +1,79 @@
+{
+    "pinVersions": false,
+    "semanticCommits": true,
+    "semanticPrefix": "build",
+    "commitMessage": "{{semanticPrefix}} update {{#if groupName}}{{{groupName}}} packages{{else}}{{{depName}}} to version {{{newVersion}}}{{/if}}",
+    "separateMajorMinor": false,
+    "prHourlyLimit": 2,
+    "labels": [
+      "target: major",
+      "action: merge"
+    ],
+    "timezone": "America/Tijuana",
+    "lockFileMaintenance": {
+      "enabled": true
+    },
+    "schedule": [
+      "after 10pm every weekday",
+      "before 4am every weekday",
+      "every weekend"
+    ],
+    "baseBranches": [
+      "master"
+    ],
+    "ignoreDeps": [
+      "@types/node",
+      "@microsoft/api-extractor"
+    ],
+    "packageFiles": [
+      "WORKSPACE",
+      "integration/bazel/WORKSPACE",
+      "package.json",
+      "packages/**/package.json",
+      "tools/ts-api-guardian/package.json",
+      "aio/package.json"
+    ],
+    "packageRules": [
+      {
+        "packagePatterns": [
+          "^@angular\/.*",
```

Escaping the `/` is not necessary (here and below): 

```suggestion
          "^@angular/.*",
```

---

Why is it not? It’s. RegExp.

---

It is a string actually :grin:
It is converted to a regex bypassing it to `RegExp`. When passing a string to `RegExp` you don't need to escape `/`, because the `RegExp` constructor knows they are part of the pattern and they are not starting/ending delimiters. Only regex literals require escaping `/`.

(Even if you needed to escape `/` in the string, you would need `\\/`. `'\/'` just escapes in the string; i.e. the resulting string is `'/'` :smiley:)